app/src/main/java/co/kr/soptandroidseminar/data/local/SharedPreference.kt

```diff
@@ -0,0 +1,47 @@
+package co.kr.soptandroidseminar.data.local
+
+import android.content.Context
+import android.content.SharedPreferences
+
+object SharedPreference {
+    private const val STORAGE_KEY = "USER_AUTH"
+    private const val AUTO_LOGIN = "AUTO_LOGIN"
+    private const val USER_ID = "USER_ID"
+    private const val USER_EMAIL = "USER_EMAIL"
+
+    fun getAutoLogin(context: Context): Boolean {
+        return getSharedPreference(context).getBoolean(AUTO_LOGIN, false)
+    }
+
+    fun getUserId(context: Context): String? {
+        return getSharedPreference(context).getString(USER_ID, "")
+    }
+
+    fun getUserEmail(context: Context): String? {
+        return getSharedPreference(context).getString(USER_EMAIL, "")
+    }
```

Comment:
```suggestion
    fun getUserId(context: Context): String {
        return getSharedPreference(context).getString(USER_ID, "") ?: ""
    }

    fun getUserEmail(context: Context): String {
        return getSharedPreference(context).getString(USER_EMAIL, "")  ?: ""
    }
```

Default 값을 주고 빈 스트링 가져오는것도 가능

Reply:
오홍 조으네요 반영완료